browser/custom_app_icons/android/BUILD.gn

```diff
@@ -101,3 +101,18 @@ java_strings_grd("java_strings_grd") {
                 android_bundle_locales_as_resources,
                 [ "values-{{source_name_part}}/android_custom_app_icons_strings.xml" ])
 }
+
+robolectric_library("custom_app_icons_junit_tests") {
```

Comment:
most of these in chrome/browser appear to be named just `junit` or `junit_tests`

Reply:
@bridiver updated the changes here : https://github.com/brave/brave-core/pull/28287/commits/32382203969524126d4035da7ad2f8544415e052